README.md

```diff
@@ -220,6 +220,10 @@ Optional variables:
     asset_cors_allow_origin: Set this to set the Access-Control-Allow-Origin for
     everything in /assets.
 
+## nodejs
+
+Used to install recent version of NodeJS. Must set `nodejs_version`. e.g. `nodejs_version: "8.x"`
```

Comment:
you should add a line explaining how to set that. not everyone is as familiar with ansible. i had to go look at nudge to remember how i did that